app/views/directives/breadcrumbs.html

```diff
@@ -1,7 +1,6 @@
 <ol class="breadcrumb" ng-if="breadcrumbs.length">
   <li ng-repeat="breadcrumb in breadcrumbs" ng-class="{'active': !$last}">
-    <a ng-if="!$last && breadcrumb.link" href="{{breadcrumb.link}}">{{breadcrumb.title}}</a>
-    <a ng-if="!$last && !breadcrumb.link" href="" back>{{breadcrumb.title}}</a>
```

Comment:
you lost this behavior for back, did you update things that were relying on that?